25-pacific_lamprey/technical_specification.md

```diff
@@ -0,0 +1,38 @@
+# Metadata refactoring proof of concept (Pacific Lamprey)
+**Epic Type:** Implementation Epic
+
+**Attention: Please do not put any confidential content here.**
+
+## Scope:
+The aim of this study is to create a proof of concept implementation of the strategy
+outlined in
+https://docs.ghga-dev.de/main/architecture_concepts/ac002_metadata_lifecycle.html.
+
+
+## Additional Implementation Details:
+
+- A implementation that can be run by data stewards locally,
+  no production-ready and independently deployed services.
+- includes:
+    - basic in implementation of the submission store
+    - Validation of metadata upon submission against the linkML schema
+    - Essential transformations and generation of associated artifacts, including:
+        - Fully embedded datasets
+        - Custom embeddings
+        - Non-restricted (public) metadata
+        - Summary statistics
+        - Mongodb-based search index
+    - REST APIs for querying the above artifacts
```

Comment:
Does this include code that serves this API or only API design? How is this to be seen in the context of a tool that is to be "run by data stewards locally"?

Reply:
As discussed, if it includes code that serves the API, then add side-loading the generated artifiacts into the database as separate point before this one.